test/parse/bad-toplevel.txt

```diff
@@ -1,13 +1,7 @@
 ;;; ERROR: 1
 (foo bar)
 (;; STDERR ;;;
-out/test/parse/bad-toplevel.txt:2:2: error: unexpected token "foo"
+out/test/parse/bad-toplevel.txt:2:1: error: unexpected token "(", expected a module field or a command.
```

Comment:
Good call; this is handled in a few other places but I think I must have missed it here. :+1: